test/integration/daemonset/util.go

```diff
@@ -27,6 +28,7 @@ import (
 var _ admission.ValidationInterface = &fakePodFailAdmission{}
 
 type fakePodFailAdmission struct {
```

Comment:
we should fix this also in the statefulset tests that do the same thing:

https://github.com/kubernetes/kubernetes/blob/2b7b5245eaff4ed60d84df67e4351b04ad9d45ae/test/integration/statefulset/util.go#L346

Reply:
Yes, it has the same issue.
Although it does not matter in the current test (with one statefulset), it would be problematic if someone makes a test running more than one statefulset.

I think this PR may be dropped as https://github.com/kubernetes/kubernetes/pull/113733 is supposed to be merged.

Maybe I need to make another PR to fix it.